mtp_api/apps/mtp_auth/tests/test_views.py

```diff
@@ -1474,6 +1536,52 @@ def test_cannot_update_prison_set_for_cashbook_user(self):
             set(current_prisons.values_list('nomis_id', flat=True))
         )
 
+    def test_updated_user_adding_fiu_group_also_adds_user_admin_group(self):
+        """
+        Test user added to FIU group also has UserAdmin group
+
+        As of MTP-1824, FIU is managing all Security users through the UserAdmin group
+        """
+        user = self.security_users[1]
+        requesting_user = self.security_uas[0]
+        current_prisons = PrisonUserMapping.objects.get_prison_set_for_user(
+            user
+        ).all()
+        user_data = {'user_admin': True}
+        self.assertUserUpdated(requesting_user, user.username, user_data)
+        updated_user = User.objects.get(username=user.username)
+        self.assertSequenceEqual(
+            updated_user.groups.order_by('name').values_list('name', flat=True),
+            ['FIU', 'Security', 'UserAdmin']
+        )
+        updated_prisons = PrisonUserMapping.objects.get_prison_set_for_user(
+            updated_user
+        ).all()
+        self.assertSequenceEqual(list(current_prisons), list(updated_prisons))
```

Comment:
do you think it's better to compare these as sets in case ordering is unstable for some reason?

Reply:
Can do, IIRC this should be a list of a single element though. My reasoning for using this is that it was what was used in the associated tests

Reply:
from what i could tell, the other list/sequence comparisons specified a sort within the test so hopefully should be good. i might well have missed some tho!